.changeset/config.json

```diff
@@ -15,5 +15,9 @@
     "@openfn/parse-jsdoc",
     "@openfn/slack-notify",
     "import-tests"
-  ]
+  ],
+  "snapshot": {
```

Comment:
hmm. My memory is tingling and I've just checked the docs. I fear we're abusing snapshots with this.

What snapshots are doing is updating the version number to include the date string. So if you do `pnpm version:snapshot`, I think you'll find that the version in package.json updates too - which we don't want